bob-common/src/mapper.rs

```diff
@@ -181,13 +180,14 @@ impl Virtual {
 
     pub fn get_operation(&self, key: BobKey) -> (VDiskId, Option<DiskPath>) {
         let virt_disk = self.get_vdisk_for_key(key).expect("vdisk not found");
-        let disk = virt_disk.replicas().iter().find_map(|disk| {
-            if disk.node_name() == self.local_node_name {
-                Some(DiskPath::from(disk))
-            } else {
-                None
-            }
-        }); //TODO prepare at start?
+        debug!("replicas: {:?}", virt_disk.replicas());
+        debug!("local_node: {:?}", self.local_node_name);
+        let disk = virt_disk
+            .replicas()
+            .get(&self.local_node_name)
+            .and_then(|replicas| {
+                replicas.get(0).and_then(|replica| Some(DiskPath::from(replica)))
```

Comment:
I think the idea of `and_then` is to call them in a chain: `.and_then(...).and_then(...)`